packages/liveblocks-core/src/crdts/LiveList.ts

```diff
@@ -134,7 +134,16 @@ export class LiveList<TItem extends Lson> extends AbstractCrdt {
     ops.push(op);
 
     for (const item of this._items) {
-      ops.push(...item._toOps(this._id, item._getParentKeyOrThrow(), pool));
+      const parentKey = item._getParentKeyOrThrow();
+      const childOps = HACK_addIntentAndDeletedIdToOperation(
```

Comment:
Had to dig up to understand what this function was doing exactly again. It adds the **set** intent and removes any potential `deletedId` value, specifically from the **first** op in the list. Maybe we can rename this function to make it a bit more clear what's happening (and why)?